packages/contracts/deploy/new/40_finalize-management-dao/00_grant-permissions.ts

```diff
@@ -28,29 +28,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     deployer
   );
 
-  // Grant `REGISTER_DAO_PERMISSION` to `Deployer`.
-  // Grant `ROOT_PERMISSION` to `PluginSetupProcessor`.
-  // Grant `APPLY_INSTALLATION_PERMISSION` to `Deployer`.
-
   const grantPermissions = [
     {
       operation: Operation.Grant,
       where: {name: 'DAORegistryProxy', address: daoRegistryAddress},
       who: {name: 'Deployer', address: deployer.address},
       permission: 'REGISTER_DAO_PERMISSION',
     },
-    {
-      operation: Operation.Grant,
-      where: {name: 'ManagementDAOProxy', address: managementDAOAddress},
-      who: {name: 'PluginSetupProcessor', address: pspAddress},
-      permission: 'ROOT_PERMISSION',
-    },
-    {
-      operation: Operation.Grant,
-      where: {name: 'PluginSetupProcessor', address: pspAddress},
-      who: {name: 'Deployer', address: deployer.address},
-      permission: 'APPLY_INSTALLATION_PERMISSION',
-    },
```

Comment:
If we still create a managing DAO, shouldn't we give the `deployer` the permissions required to install plugins on it already?

Otherwise, the deployer needs to execute more transactions himself.
I guess this will change in https://aragonassociation.atlassian.net/browse/OS-1236